File: Resueltos_por_competidor/Miguel_Lopez/Drinks.cpp
/*
 * Autor: Miguel Lopez
 * Problema:  Drinks
 * Juez online: Codeforces
 * Veredicto: Accepted
 * Url: https://codeforces.com/problemset/problem/200/B
 */

#include <bits/stdc++.h>
#include <cctype>
#include <cstdio>
#include <iomanip>
#include <ios>
typedef long long int ll;
using namespace std;
void init_code() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    #ifndef ONLINE_JUDGE
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
    #endif 
}
int main() {
   init_code();
   int t; cin >> t;
   double c = 0;
   for(int i = 0; i < t ; i++) {
      double  n; cin >> n;
      c+=(n/t);
   }  
   cout << fixed << setprecision(12) << c <<  endl; 
   return 0;
}



 

File: Resueltos_por_competidor/Jacobo_Segura/Drinks.java
/* 
Codeforces 200B - Drinks
https://codeforces.com/problemset/problem/200/B
*/
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		double cont = 0;
		for(int i = 0; i<n; i++) {
			int p = sc.nextInt();
			cont += p;
		}
		System.out.println(cont/n);
	}
}
